pkg/core/options.go

```diff
@@ -30,49 +32,56 @@ var (
 )
 
 // ConcurrentList sets the max level of concurrency to retrieve core objects. It defaults to 2 x #cpus.
-func ConcurrentList(concurrentList int) Option {
+func ConcurrentList(v int) Option {
 	return func(s *Settings) {
-		if concurrentList == 0 {
+		if v == 0 {
```

Comment:
the general idea when renaming identifiers was to be brief, not terse. This is just code obfuscation. I sense some irony behind this change...

Reply:
no gaslighting on this end of the wire:

this _was_ motivated by my understanding of your preferences for terser names.  it is _not_ intended to be ironic.  the `v` for "value" name is rather an attempt for consistency with [the convention for type assertions](https://github.com/ransomw/exercises/blob/master/gotour/mni/main.go#L231) in the golang tour.